index.php

```diff
@@ -150,12 +154,12 @@ function get_mediatype_sql($m)
                     // make sure filter is valid
                     if (!array_key_exists($filter, $filter_expr)) $filter = 'ABC';
                     // apply filter
-                    $WHERES .= ' AND title RLIKE \''.utf8_encode($filter_expr[$filter]).'\'';# AND mediatype != '.MEDIA_WISHLIST;
+                    $WHERES .= ' AND title RLIKE \''.mb_convert_encoding($filter_expr[$filter], 'UTF-8', 'ISO-8859-1').'\'';# AND mediatype != '.MEDIA_WISHLIST;
 }
 
-if(!$ORDER)
+if(!isset($ORDER))
```

Comment:
Suggest changing this to `if (!(isset($ORDER) && $ORDER))`

Reply:
i believe that new code is ok. 
if $ORDER is not set then need code to be executed to setup $ORDER